xa/nodejs/department/readme.md

```diff
@@ -33,12 +30,15 @@ Set database connection parameters in `dbconfig.ts` file.
 export default {
     user          : process.env.DEPARTMENTDATASOURCE_USER  || "<user-name>",
     password      : process.env.DEPARTMENTDATASOURCE_PASSWORD || "<db-password>",
-    connectString : process.env.DEPARTMENTDATASOURCE_URL || "<connection-string>"
+    connectString : process.env.DEPARTMENTDATASOURCE_URL || "<connection-string>",
+    walletLocation: process.env.DEPARTMENTDATASOURCE_WALLETLOCATION || "<wallet_location",
+    walletPassword: process.env.DEPARTMENTDATASOURCE_WALLETPASSWORD || "<wallet_password>"
+    
```

Comment:
Note: If the connecting database uses wallet, `walletLocation` and `walletPassword` property needs to be set.